packages/codecs-core/src/__benchmarks__/run.ts

```diff
@@ -4,9 +4,9 @@ import { webcrypto as crypto } from 'node:crypto';
 
 import { Bench } from 'tinybench';
 
-import { createCodec, FixedSizeCodec } from '../codec';
-import { ReadonlyUint8Array } from '../readonly-uint8array';
-import { reverseCodec } from '../reverse-codec';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
These ignore directives imply that these benchmarks will break frequently without warning, but I don't know of any other way to:

1. Refer to the built source
2. Before it exists
3. In a way that will typecheck

Reply:
I figured it out.